cpp/dolfinx/io/XDMFFile.cpp

```diff
@@ -321,6 +321,7 @@ mesh::MeshTags<std::int32_t>
 XDMFFile::read_meshtags(const std::shared_ptr<const mesh::Mesh>& mesh,
                         const std::string name, const std::string xpath)
 {
+  LOG(INFO) << "XDMF read meshtags (" << name << ")";
```

Comment:
Does `xdmf_utils::distribute_entity_data` below just read, or is it doing a lot of distribution work? If it's doing a lot of work it would be helpful to add another log message just before it.

Reply:
It does a lot of redistribution, but there are further log messages inside it now, before each MPI call.